<launch>
  <!-- 1) Start Gazebo simulation with your world -->
  <include file="$(find tiago_iaslab_simulation)/launch/start_simulation.launch">
    <arg name="world_name" value="iaslab_assignment2"/>
  </include>

  <!-- 2) Bring up the Navigation stack -->
  <include file="$(find tiago_iaslab_simulation)/launch/navigation.launch"/>

  <!-- 3) Your straight-line service node -->
  <node pkg="tiago_iaslab_simulation"
        type="get_straightline_node"
        name="get_straightline_node"
        output="screen"/>

  <!-- 4) Our new collision‐object publisher -->
  <node pkg="assignment2_package"
        type="test_node"
        name="test_node"
        output="screen"/>
</launch>
